bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClientImpl.java

```diff
@@ -467,7 +474,7 @@ public void readLac(final BookieSocketAddress addr, final long ledgerId, final R
             } else {
                 pcbc.readLac(ledgerId, cb, ctx);
             }
-        }, ledgerId);
+        }, ledgerId, conf.getUseV2WireProtocol());
```

Comment:
I don't think the value here should be `conf.getUseV2WireProtocol()`. It should be `true /* forceUseV3 *`, right?